lesson03/assignment/css/style.scss

```diff
@@ -3,4 +3,131 @@
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
+@import 'body.set';
 
+/* set card width, border, etc. */
+.card {
+  display: block;
+  max-width:60rem;
+  border:2rem solid $gray;
+  background-color: $white;
+  padding: 2rem;
+  .card-body {
+    display: flex;
+    flex-direction: row;
+    background-color:$white;
+  }
+}
+/* set up card-copy divs */
+.card-copy {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* first column needs margin to the right */
+  &:first-child {
+    margin-right: 2rem;
+  }
+  /* second column needs background color gray
+    and needs to grow to fill the space  */
+  &:last-child {
+    background-color: $gray;
+    flex-grow:1;
+  }
+  /* remove bullets, increase padding increase line-height */
+  .about {
+    list-style: none;
+    padding-left:2rem;
+    .item {
+      margin-bottom: 1rem;
+    }
+    /* align text with center of image and add padding */
+    .linkedin {
+      display: flex;
+      align-items: center;
+      img {
+        margin-right:.5rem;
+      }
+    }
+  }
+  /* style card title, name and job */
+  .card-title {
+    .name {
+      margin-bottom:0;
+      font-size:1.625rem;
+    }
+    .job {
+      margin-top:0;
+      font-style: italic;
+    }
+  }
+  /* set up image div */
+  .card-img {
+    display: flex;
+    padding-top:6rem;
+    align-items: center;
+    justify-content: center;
+    max-width:100%;
+    border-radius: 100%;
+    overflow: hidden;
+    width: 14rem;
+    height: 14rem;
+    overflow: hidden;
+    /* align image within div */
+    img {
+      max-width: 150%;
+      margin-left: auto;
+      margin-right: auto;
+    }
+  }
+}
+
+/* // Medium devices (tablets, 768px and under) */
+@include media-query("mobile") {
+  .card {
+    /* display in column to save space */
+    .card-body {
+      flex-direction: column;
+      justify-content: center;
+    }
+  }
```

Comment:
If there are no styles for `.card`, only its child `.card-body`, there's no need to nest. It can just be a selector of `.card .card-body` as you would use in plain CSS.

Reply:
For practice: I made this change, I asked myself this question exactly as I was writing the code. Thank you!